gtp5g wont build on ubuntu 20.04.5 kernel 5.4.0-050400-generic with 'pktinfo.c:84:8: error: too many arguments to function ‘rt->dst.ops->update_pmtu’'

Hello:

I'm trying to build and install gtp5g on my 20.04.5 Ubuntu VM with kernel 5.4.0-050400-generic but am hitting the following error. Please help me understand if this is known and if there are any workarounds or fixes.

```
root@rigel:~# hostnamectl
   Static hostname: rigel
         Icon name: computer-vm
           Chassis: vm
        Machine ID: a8ad6bb1840c4ced8a58115da013d9de
           Boot ID: bf5d204c7f244510ba845dd97f9108f9
    Virtualization: kvm
  Operating System: Ubuntu 20.04.5 LTS
            Kernel: Linux 5.4.0-050400-generic
      Architecture: x86-64
root@rigel:~# cd gtp5g/
root@rigel:~/gtp5g# make clean && make
make -C /lib/modules/5.4.0-050400-generic/build M=/root/gtp5g clean
make[1]: Entering directory '/usr/src/linux-headers-5.4.0-050400-generic'
  CLEAN   /root/gtp5g/Module.symvers
make[1]: Leaving directory '/usr/src/linux-headers-5.4.0-050400-generic'
make -C /lib/modules/5.4.0-050400-generic/build M=/root/gtp5g modules
make[1]: Entering directory '/usr/src/linux-headers-5.4.0-050400-generic'
  CC [M]  /root/gtp5g/src/gtp5g.o
  CC [M]  /root/gtp5g/src/log.o
  CC [M]  /root/gtp5g/src/util.o
  CC [M]  /root/gtp5g/src/gtpu/dev.o
  CC [M]  /root/gtp5g/src/gtpu/encap.o
  CC [M]  /root/gtp5g/src/gtpu/hash.o
  CC [M]  /root/gtp5g/src/gtpu/link.o
  CC [M]  /root/gtp5g/src/gtpu/net.o
  CC [M]  /root/gtp5g/src/gtpu/pktinfo.o
/root/gtp5g/src/gtpu/pktinfo.c: In function ‘ip4_find_route’:
/root/gtp5g/src/gtpu/pktinfo.c:84:8: error: too many arguments to function ‘rt->dst.ops->update_pmtu’
   84 |        rt->dst.ops->update_pmtu(&rt->dst, NULL, skb, mtu, false);
      |        ^~
make[2]: *** [scripts/Makefile.build:266: /root/gtp5g/src/gtpu/pktinfo.o] Error 1
make[1]: *** [Makefile:1652: /root/gtp5g] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-5.4.0-050400-generic'
make: *** [Makefile:79: module] Error 2
root@rigel:~/gtp5g#
```

--- a/src/gtpu/pktinfo.c
+++ b/src/gtpu/pktinfo.c
@@ -80,7 +80,7 @@ struct rtable *ip4_find_route(struct sk_buff *skb, struct iphdr *iph,
         mtu = dst_mtu(&rt->dst);
     }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0) || defined(RHEL8)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 8) || defined(RHEL8)
        rt->dst.ops->update_pmtu(&rt->dst, NULL, skb, mtu, false);
 #else
        rt->dst.ops->update_pmtu(&rt->dst, NULL, skb, mtu);

